component/portal/src/main/java/org/exoplatform/portal/rest/GroupRestResourcesV1.java

```diff
@@ -42,6 +42,8 @@ public class GroupRestResourcesV1 implements ResourceContainer {
   public static final int     DEFAULT_LIMIT  = 20;
 
   public static final int     DEFAULT_OFFSET = 0;
+  public static final String     SPACES_GROUP_TYPE = ".spaces";
```

Comment:
Same for spaces, IMO it's better to not embed this in REST business logic which has to be without business logic (See Software Architecture Guidelines).

Reply:
I know, but we have a specific need here, and I don't want to add a new method to the organization service on gatein for this specific need from social, that is why I put it here.